### ISSUE TITLE
Display wallet's accounts in the address book

### ISSUE THREAD

**Issue by OlegMakarenko** [nemgrouplimited/symbol-desktop-wallet#1098]
![image](https://user-images.githubusercontent.com/33131259/107583057-0df82500-6c03-11eb-9e75-88742628ee72.png)

**Comment by myxmaster** [comment]
Yes, I think all the accounts from current wallet (own accounts) should be suggested in the "Select contact" list.

To prevent confusion we should forbid to add own accounts as contacts, because then we have possible account name collisions.

**Comment by OlegMakarenko** [comment]
I agree. I see there are two ways: 
1. Display all wallet accounts in the contact dropdown by default and disallow adding them to the address book. Probably dropdown should have two groups (owned accounts and address book).

2. Add checkbox "Add this account to address book" to the account creation modal. When user rename or delete an account there should be a modal which propose to update address book

**Comment by myxmaster** [comment]
1. is way better in my opinion
